configs/milvus.yaml

```diff
@@ -225,6 +225,13 @@ grpc:
     keepAliveTime:    10000
     keepAliveTimeout: 3000
 
+# Configure the proxy tls enable.
+#tls:
```

Comment:
We'd better uncomment tls configs,  use a bool value to enable tls that is similar to authorizationEnabled(277 line) under security